aip/auth/4110.md

```diff
@@ -0,0 +1,153 @@
+---
+aip:
+  id: 4110
+  scope: auth
+  state: approved
```

Comment:
I usually recommend committing as reviewing first and having a separate approval step with a wider audience. Up to you though.

Reply:
Changed to reviewing.